litmus-portal/backend/graphql-server/pkg/database/mongodb/project.go

```diff
@@ -0,0 +1,38 @@
+package database
+
+import (
+	"context"
+	"log"
+
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/graph/model"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var projectCollection *mongo.Collection
```

Comment:
All the collection variables are in the init.go file

Reply:
Yes but I think having all the collection variables in their separate files will be better for the code to be easily enhanced in the near future. We will just have to create new files for collections and do the minimal editing of already created files. 
Instead, I think we should have the name of the collections in these files too as it will not require the init file to be edited every time if we need to have more collections which we surely will need.